Update questions.yaml

Kimono8137 opened the pull request:
updated Pool, Wallet & Worker variables to properly apply changes to config file

**Description**
<!--
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
-->
⚒️ Fixes  # <!--(issue)-->

**⚙️ Type of change**

- [ ] ⚙️ Feature/App addition
- [ ] 🪛 Bugfix
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 🔃 Refactor of current code

**🧪 How Has This Been Tested?**
<!--
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
-->

**📃 Notes:**
<!-- Please enter any other relevant information here -->

**✔️ Checklist:**

- [ ] ⚖️ My code follows the style guidelines of this project
- [ ] 👀 I have performed a self-review of my own code
- [ ] #️⃣ I have commented my code, particularly in hard-to-understand areas
- [ ] 📄 I have made corresponding changes to the documentation
- [ ] ⚠️ My changes generate no new warnings
- [ ] 🧪 I have added tests to this description that prove my fix is effective or that my feature works
- [ ] ⬆️ I increased versions for any altered app according to semantic versioning

**➕ App addition**

If this PR is an app addition please make sure you have done the following.

- [ ] 🖼️ I have added an icon in the Chart's root directory called `icon.png`

---

_Please don't blindly check all the boxes. Read them and only check those that apply.
Those checkboxes are there for the reviewer to see what is this all about and
the status of this PR with a quick glance._

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/truecharts/charts?pullRequest=18491) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/truecharts/charts?pullRequest=18491) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/truecharts/charts?pullRequest=18491) it.</sub>

xstar97 commented on charts/stable/xmrig/questions.yaml:
```diff
@@ -13,19 +13,19 @@ questions:
                                     attrs:
-                                      - variable: POOL
+                                      - variable: POOL_URL
                                         label: "POOL"
```
```suggestion
                                        label: "Pool Url"
```

xstar97 commented on charts/stable/xmrig/questions.yaml:
```diff
@@ -13,19 +13,19 @@ questions:
                                           type: string
                                           default: "xmr-au1.nanopool.org:14444"
-                                      - variable: WALLET
+                                      - variable: YOUR_ADDRESS
                                         label: "WALLET"
                                         description: "Wallet address for receiving mined coin. If you are privacy conscious, you can redact this in logs posted to the support thread by replacing it with WALLET. If you use Monero, sharing this address will only allow people to send you Monero and possibly identify you if you send someone else Monero, but being a privacy oriented coin its entirely  ...
                                         schema:
                                           type: string
                                           default: "84e8UJvXHDGVfE5HZDQfhn3Kh3RGJKebz31G7D4H24TLPMe9x7bQLBw8iyBhNx9USXB8MhvhBe3DyVW1LcuVAf4jBiADNLw"
```
```suggestion
                                      - variable: YOUR_ADDRESS
                                        label: "Wallet Address"
                                        description: "Wallet address for receiving mined coin."
                                        schema:
                                          type: string
                                          private: true
                                          default: ""
```

xstar97 commented on charts/stable/xmrig/questions.yaml:
```diff
@@ -13,19 +13,19 @@ questions:
                                           type: string
                                           default: "84e8UJvXHDGVfE5HZDQfhn3Kh3RGJKebz31G7D4H24TLPMe9x7bQLBw8iyBhNx9USXB8MhvhBe3DyVW1LcuVAf4jBiADNLw"
-                                      - variable: WORKER
+                                      - variable: YOUR_WORKER_NAME
                                         label: "WORKER"
                                         description: "Enter a name for your worker. Some pools use a hidden option with this (Nanopool included) that allows you to change your payment threshold on their website. If you want to use this option, you can change this to workernameemail, eg. TrueNAStest@example.com, where test@example.com is your own email address. This option is basically a password. ...
```
```suggestion
                                      - variable: YOUR_WORKER_NAME
                                        label: "Worker Name"
                                        description: "Enter a name for your worker."
```

xstar97 reviewed (changes requested):
patch bump the chart version (NOT the app version

Kimono8137 commented:
> patch bump the chart version (NOT the app version

Sorry, do you mean on the linked issue? I'm just knowledgable enough to be clueless!

Ornias1993 reviewed (changes requested):
Use correct semver PR titles please.

StevenMcElligott commented:
Also need to update the `values.yaml`, not just the questions ;-)

Kimono8137 commented:
> Also need to update the `values.yaml`, not just the questions ;-)

I made the changes to the values, hopefully I did it correctly. I can't figure out how to push it to this PR though.

Ornias1993 reviewed (changes requested):
Needs a version increase and all env-vars should (also) be reflected in values.yaml

Kimono8137 commented:
> Needs a version increase and all env-vars should (also) be reflected in values.yaml

Sorry, I see what I missed. I bumped the chart version and updated env vars in values.yaml

xstar97 commented:
https://xmrig.com/docs/algorithms
Is COIN still an option?
```ALGO_MODE```
Default: rx/0

Kimono8137 commented:
> https://xmrig.com/docs/algorithms Is COIN still an option? `ALGO_MODE` Default: rx/0

If I'm reading it correctly, the docker image the chart is based on doesn't use COIN variable, just ALGO_MODE [https://hub.docker.com/layers/cooleye72/xmrig/latest/images/sha256-5ad4b674874a4024462d320a823b7f361129875f2d8616e4e1bb26e8169f5409?context=explore](url) I've updated the values.yaml and questions.yaml accordingly. I also removed the Donation, FEE and DRIVERV variables as they aren't in the docker image.
